### DISCUSSION TITLE
Add page similarity comparison function

### DISCUSSION THREAD

**Discussion by evilc0deooo** [projectdiscovery/httpx#1250] [Q&A]
Add page similarity comparison function to improve the accuracy of path scanning and reduce false positives，and -fs, -filter-string String filter responses with specified strings, add support for importing string files, in order to support a large number of target scans, need to filter multiple strings. Thanks

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
